.github/workflows/build.yml

```diff
@@ -149,6 +149,11 @@ jobs:
         run: |
           cd build/bin
           ./test_spdm_crypt
+      - name: Test_crypt
```

Comment:
`test_crypt` always returns success to CI/CD.
```
int main(void)
{
    libspdm_cryptest_main();
    return 0;
}
```

Reply:
I have submitted a new PR #1350 to fix it.